Doppler.Sap.Job/Database/DopplerRepository.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System.Collections.Generic;
+using System.Data.SqlClient;
+using System.Threading.Tasks;
+using Dapper;
+
+namespace Doppler.Sap.Job.Service.Database
+{
+    public class DopplerRepository : IDopplerRepository
+    {
+        private readonly string _connectionString;
+
+        public DopplerRepository(string connectionString)
```

Comment:
How does DI container to know what string inject?

Reply:
I changed this code in this commit https://github.com/MakingSense/doppler-jobs/pull/6/commits/f5d1f949ed4d5c0dff5e14b70938148ea69b8c86